recipes/occt/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: occt
+  version: 7.0.0
+
+source:
+  fn: occt.tar.gz
+  url: http://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=1d505bb0f8a64feb87c76a230b64168242461d94;sf=tgz
```

Comment:
Well that is the other part. We should really be downloading a tarball. Can we look into that more?